make/compiler/Makefile.gnu

```diff
@@ -120,10 +120,18 @@ ifndef GNU_GCC_MINOR_VERSION
 export GNU_GCC_MINOR_VERSION := $(shell $(CC) -dumpversion | awk '{split($$1,a,"."); printf("%s", a[2]);}')
 endif
 ifndef GNU_GPP_MAJOR_VERSION
-export GNU_GPP_MAJOR_VERSION := $(shell $(CXX) -dumpversion | awk '{split($$1,a,"."); printf("%s", a[1]);}')
+ifneq ($(MAKE_LAUNCHER),1)
```

Comment:
Please add a comment here about what's going on (covering it for the first ifneq launcher is sufficient)